docs/integration-guides/advanced.md

```diff
@@ -449,7 +449,11 @@ To subscribe to all votes notifications:
 
 ##### Filtering options
 
-Filters for **votes** can be used to subscribe only to votes from selected representatives. Once filters are given, votes from representatives that do not match the options are not broadcasted.
+The following filtering options can be combined.
+
+###### Representatives
+
+Used to subscribe only to votes from selected representatives. Once filters are given, votes from representatives that do not match the options are not broadcasted. If the result is an empty filter (for example, all given accounts are invalid), then the filter is not used. A message is logged in the node logs when this happens.
```

Comment:
This is unique to V21+ right?

Reply:
With this websocket being marked as debug I thought making it simpler by not including the differences would be easier. Especially since the pre-V21 is not reliable for all types of votes.